sdk/synapse/Azure.Analytics.Synapse.AccessControl/tests/AccessControlClientLiveTests.cs

```diff
@@ -20,116 +19,116 @@ namespace Azure.Analytics.Synapse.AccessControl.Tests
-                InstrumentClientOptions(new AccessControlClientOptions())
-            ));
-        }
-
-        [Test]
-        public async Task CreateRoleAssignment()
-        {
-            RoleAssignmentsClient assignmentsClient = CreateAssignmentClient();
-            RoleDefinitionsClient definitionsClient = CreateDefinitionsClient();
-
-            await using DisposableClientRole role = await DisposableClientRole.Create (assignmentsClient, definitionsClient, TestEnvironment);
-
-            Assert.NotNull(role.Assignment.Id);
-            Assert.NotNull(role.Assignment.RoleDefinitionId);
-            Assert.NotNull(role.Assignment.PrincipalId);
-        }
-
-        [Test]
-        public async Task GetRoleAssignment()
-        {
-            RoleAssignmentsClient assignmentsClient = CreateAssignmentClient();
-            RoleDefinitionsClient definitionsClient = CreateDefinitionsClient();
-
-            await using DisposableClientRole role = await DisposableClientRole.Create (assignmentsClient, definitionsClient, TestEnvironment);
-
-            RoleAssignmentDetails roleAssignment = await assignmentsClient.GetRoleAssignmentByIdAsync(role.Assignment.Id);
-
-            Assert.AreEqual(role.Assignment.RoleDefinitionId, roleAssignment.RoleDefinitionId);
-            Assert.AreEqual(role.Assignment.PrincipalId, roleAssignment.PrincipalId);
-        }
-
-        [Test]
-        public async Task ListRoleAssignments()
-        {
-            RoleAssignmentsClient assignmentsClient = CreateAssignmentClient();
-            RoleDefinitionsClient definitionsClient = CreateDefinitionsClient();
-
-            await using DisposableClientRole role = await DisposableClientRole.Create (assignmentsClient, definitionsClient, TestEnvironment);
-
-            Response<IReadOnlyList<SynapseRoleDefinition>> roleAssignments = await definitionsClient.ListRoleDefinitionsAsync();
-            foreach (SynapseRoleDefinition expected in roleAssignments.Value)
-            {
-                SynapseRoleDefinition actual = await definitionsClient.GetRoleDefinitionByIdAsync(expected.Id.ToString());
-                Assert.AreEqual(expected.Id, actual.Id);
-                Assert.AreEqual(expected.Name, actual.Name);
-            }
-            Assert.GreaterOrEqual(roleAssignments.Value.Count, 1);
-        }
-
-        [Test]
-        public async Task DeleteRoleAssignments()
-        {
-            RoleAssignmentsClient assignmentsClient = CreateAssignmentClient();
-            RoleDefinitionsClient definitionsClient = CreateDefinitionsClient();
-
-            RoleAssignmentDetails assignment = await DisposableClientRole.CreateResource (assignmentsClient, definitionsClient, TestEnvironment);
-
-            Response response = await assignmentsClient.DeleteRoleAssignmentByIdAsync (assignment.Id);
-            response.AssertSuccess();
-        }
-    }
+    //[LiveOnly] // Assignment IDs can not be reused for at least 30 days.
```

Comment:
Tests will be restored in: https://github.com/azure/azure-sdk-for-net/issues/24394